python/src/trezorlib/cli/cardano.py

```diff
@@ -139,6 +160,8 @@ def get_address(
     block_index,
     tx_index,
     certificate_index,
+    script_payment_hash,
+    script_staking_hash,
```

Comment:
What the `staking_address` field above? Sounds suspicious.

Reply:
Yeah, the payment address path is called just `address` from before Shelley. It's a Trezor convention similar to the paths being called `address_n` and `address_n_staking`. I'm not sure if we can do anything about it now.